src/app/(user)/profile.tsx

```diff
@@ -211,8 +211,8 @@ export default function Profile(): JSX.Element {
                 style={{
```

Comment:
When editing the styles, please move them to StyleSheets as it's our aim to completely transition to them.